sig-security-tooling/cve-feed/README.md

```diff
@@ -0,0 +1,48 @@
+# Official CVE Feed
+
+The official CVE feed is separated into two main components:
+1. the scripts, updating a cloud bucket containing the feed;
+2. the website rendering and serving the feed to various formats.
+
+## Scripts
+
+A script in the [kubernetes/sig-security](https://github.com/kubernetes/sig-security)
+repository under the [sig-security-tooling/cve-feed/hack](https://github.com/kubernetes/sig-security/tree/main/sig-security-tooling/cve-feed/hack)
+folder. This script is
+a bash script named `fetch-cve-feed.sh` that:
+- setups the python3 environment;
+- generates the CVE feed file with `fetch-official-cve-feed.py`;
+- compares the sha256 of the newly generated file with the existing one;
+- if the sha256 changed, uploads the newly generated CVE feed file to the bucket.
+
+The `fetch-official-cve-feed.py` file executed by the `fetch-cve-feed.sh` is a
+python3 script that:
+- queries the GitHub API to fetch all the issues with the `official-cve-feed`
+  label in the [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue%20label%3Aofficial-cve-feed%20)
+  repository;
+- formats the result with the appropriate JSON schema to be JSON feed
+  compliant;
+- prints the output to stdout.
+
+These scripts are run regularly as a CronJob on the k8s infrastructure.
+
+In short, these scripts take the GitHub [kubernetes/kubernetes issues
+labeled with `official-cve-feed`](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue%20label%3Aofficial-cve-feed%20)
+as the input and generate a JSON feed file as an output in a cloud bucket. The
+output can be publicly fetched at [gs://k8s-cve-feed/](https://console.cloud.google.com/storage/browser/k8s-cve-feed) or [storage.googleapis.com/k8s-cve-feed](https://storage.googleapis.com/k8s-cve-feed/).
```

Comment:
is `gs://k8s-cve-feed/` a typo?

Reply:
This is the usual way to format a URL for a google cloud storage bucket